frontend/device/remarkable/device.lua

```diff
@@ -153,8 +151,23 @@ function Remarkable:init()
     local scalex = screen_width / self.mt_width
     local scaley = screen_height / self.mt_height
 
-    self.input:registerEventAdjustHook(adjustAbsEvt)
-    self.input:registerEventAdjustHook(self.adjustTouchEvent, {mt_scale_x=scalex, mt_scale_y=scaley})
+    -- Assume input stuff is saner on mainline kernels...
+    -- (c.f., https://github.com/koreader/koreader/issues/10012)
+    local is_mainline = false
+    --- @fixme: Find a better way to discriminate mainline from stock...
+    local std_out = io.popen("uname -r", "r")
+    if std_out then
+        local release = std_out:read("*line")
+        release = release:match("^(%d+%.%d+)%.%d+.*$")
```

Comment:
Maybe part of todo but what exactly does mainline vs not look like in this case? Or put another way, what's it testing for?

Reply:
rM currently uses an older kernel release (based on NXP's tree).

Ideally, I'd like to check something *other* than the release, because rM happens to be the *one* vendor that appears to care about keeping its software stack up-to-date, so this is very much not future-proof ;).

Reply:
But, as I don't own any of these devices, this is an exercise best left to someone who has access to one ;o).

Reply:
Ohh I thought I had to be missing but it does what it looks like it does. Gotcha.